literature.bib

```diff
@@ -24908,6 +25036,19 @@ @book{S01
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2000 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{Wells00,
+	renamedFrom = {BusyBox},
+    title={{Busybox: A Swiss Army Knife for Linux}},
+	author={Wells, Nicholas},
+	journal={Linux Journal},
+	volume={2000},
+	number={78es},
+	year=2000,
+	publisher={Belltown Media Houston, TX},
+    doi={10.5555/364412.364422},
+    howpublished = {Website: \url{https://busybox.net/}}
```

Comment:
Should be either _eprint_ or _note_ for article
Since the URL does not link to the article, I would go for `note = {Website: \url{https://busybox.net/}`

Reply:
done